FirebaseSegmentation/Tests/Unit/SEGDatabaseManagerTests.m

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "SEGDatabaseManager.h"
+
+#import <OCMock/OCMock.h>
+
+@interface SEGDatabaseManager (Test)
+- (NSString *)pathForSegmentationDatabase;
+@end
+
+@interface SEGDatabaseManagerTests : XCTestCase {
+  long _expectationTimeout;
+}
+
+@end
+
+@implementation SEGDatabaseManagerTests
+
+- (void)setUp {
+  // Override the database path to create a test database.
+  id databaseManagerMock = OCMClassMock([SEGDatabaseManager class]);
+  OCMStub([databaseManagerMock pathForSegmentationDatabase])
+      .andReturn([self pathForSegmentationTestDatabase]);
+
+  // Expectation timeout for each test.
+  _expectationTimeout = 2;
+}
+
+- (void)tearDown {
+  // Put teardown code here. This method is called after the invocation of each test method in the
+  // class.
```

Comment:
The same as for `SEGContentManagerTests` - tear down mocks

Reply:
Done.